conf/ocsci/production_powervs_upi.yaml

```diff
@@ -0,0 +1,30 @@
+RUN:
+  bin_dir: '/usr/local/bin'
+
+DEPLOYMENT:
+  local_storage: True
+  force_download_installer: False
+  force_download_client: False
+  ocs_csv_channel: "stable-4.6"
+  default_latest_tag: "latest-4.6"
+ENV_DATA:
+  storage_device_sets_name: "ocs-deviceset"
+  ocs_version: '4.6'
+  skip_ocp_deployment: true
```

Comment:
this should not be here, but rather used as a run-ci `--ocsci-conf` param when executing

Reply:
@ebenahar you should instead use `--ocs-version 4.6`, the [README](https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocs_version/README.md) explains why.

Reply:
I will use --ocs-version 4.6.

Reply:
Done.